Core/CoreTests/Conferences/ConferenceDetailsViewControllerTests.swift

```diff
@@ -41,6 +46,11 @@ class ConferenceDetailsViewControllerTests: CoreTestCase {
         ]))
     }
 
+    override func tearDown() {
+        Clock.reset()
+        super.tearDown()
+    }
+
```

Comment:
Thought about it, but since we have test classes not subclassing CoreTestCase (I assume they don't need the extra setup work) it would give a false safety. I would hope to rely on making it a habbit to reset everywhere, consistently (also to help local reasoning)

Reply:
Sorry, misunderstood, I thought you suggested to put it in `tearDown()`.
Good point, will put it into `CoreTestCase.setUp()`